pkgs/by-name/le/lean4/package.nix

```diff
@@ -8,29 +8,46 @@
   cadical,
   pkg-config,
   libuv,
+  use-mimalloc ? true,
   perl,
   testers,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "lean4";
-  version = "4.18.0";
+  version = "4.19.0";
+
+  mimalloc-src = fetchFromGitHub {
```

Comment:
Is there a good reason to use a vendored version instead of using mimalloc from nixpkgs?